Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -5624,6 +5624,7 @@ void LocalFrameView::removeScrollableAreaForAnimatedScroll(ScrollableArea* scrol
 
 bool LocalFrameView::addScrollableArea(ScrollableArea* scrollableArea)
 {
+    ASSERT(scrollableArea);
```

Comment:
Rather than add this assertion, it would be better to change the argument to a `ScrollableArea&`.

Reply:
Let me try on local build. :-)